src/theme/theme.ts

```diff
@@ -54,6 +59,11 @@ export const theme: DefaultTheme = {
         default: colors.mercuryGrey,
         hover: colors.manuscriptsLight,
       },
+      newDesign: {
```

Comment:
Please change the name to something more meaningful

Reply:
changed to 'tracked'

Reply:
Great, thanks